src/main/java/racing/domain/Car.java

```diff
@@ -3,18 +3,45 @@
 public class Car {
 
   private static final int MOVE_THRESHOLD = 4;
+  private static final int MAX_NAME_LENGTH = 5;
+  private static final int MIN_RANDOM_VALUE = 0;
+  private static final int MAX_RANDOM_VALUE = 9;
 
   private int position;
 
+  private String name;
```

Comment:
```suggestion
  private final String name;
```
name은 변경되지 않으므로 불변하게 수정 가능하겠네요!